packages/ed25519-signature-2018/src/__tests__/getVerificationMethod.test.ts

```diff
@@ -28,9 +39,19 @@ describe("document loader testing", () => {
     const verifiedProof = await suite.verifyProof({
       proof,
       document: document,
-      purpose: new purposes.AssertionProofPurpose(),
+      purpose: {
```

Comment:
consider writing this function in 1 place, and importing it, instead of repeating yourself like this.